arbeitszeit_flask/views/request_coordination_transfer_view.py

```diff
@@ -24,7 +24,6 @@ class RequestCoordinationTransferView:
     controller: RequestCoordinationTransferController
     use_case: RequestCoordinationTransferUseCase
 
-    @commit_changes
```

Comment:
I think there might be another problem here with templates/user/coop_summary.html" — another link in the form of a button for transferring the coordination.  It's line 37 in master.

Reply:
You mean this, right?

```python
{% if view_model.transfer_coordination_url is not none %}
      <a class="button is-small is-primary" href="{{ view_model.transfer_coordination_url }}">
          <span class="icon">
              {{ "square-arrow-up-right"|icon }}
          </span>
          <span>{{ gettext("Transfer coordination") }}</span>
      </a>
 {% endif %}
  ```
  
  This link does lead to the `request_coordination_transfer` route in the company blueprint. But it leads to the GET method of this view and there is no state change implied, so there should be no problem.

Reply:
Let me know if this can get merged from your point of view.